src/models/KeyboardModel.ts

```diff
@@ -97,7 +98,11 @@ class KeymapItem {
   }
 
   get isDefault(): boolean {
-    return this.option == OPTION_DEFAULT;
+    return this.option == OPTION_DEFAULT || this.choice == '0';
```

Comment:
```suggestion
    return this.option == OPTION_DEFAULT || this.isOrigin;
```

I think that the above code will become more understandable code.